src/set_relation/set_relation.cc

```diff
@@ -2944,5 +2944,79 @@ Relation* Relation::addConstraintsDueToMonotonicity() const {
     return retval;
 }
 
+/*****************************************************************************/
+#pragma mark -
+/*************** setExpType **************************************************/
+
+void Conjunction::setExpType()
```

Comment:
Don't need this method.  Just set the expression type in Conjunction::addInequality and Conjunction::addEquality